tests/e2e/specs/drawing/profile.spec.js

```diff
@@ -75,57 +76,46 @@ describe('Profile popup', () => {
         })
-        it('shows the popup again (maximized) when clicking on the header while minimized', () => {
-            cy.get('[data-cy="profile-popup-header"]').click().click()
-            cy.get('[data-cy="profile-popup"]').should('be.visible')
-            cy.get('[data-cy="profile-popup-content"]').should('be.visible')
+        it('minimizes and restores the profile popup when clicking on the header', () => {
+            cy.get('[data-cy="infobox-header"]').should('be.visible')
+            cy.get('[data-cy="infobox-content"]').should('be.visible')
+
+            cy.get('[data-cy="infobox-header"]').click()
+            cy.get('[data-cy="infobox-header"]').should('be.visible')
+            cy.get('[data-cy="infobox-content"]').should('not.be.visible')
+
+            cy.get('[data-cy="infobox-header"]').click()
+            cy.get('[data-cy="infobox-header"]').should('be.visible')
+            cy.get('[data-cy="infobox-content"]').should('be.visible')
         })
-        it('closes the profile popup when the user clicks somewhere else on the map', () => {
+        it('closes the popup when clicking elsewhere on the map', () => {
             cy.get(olSelector).click(360, 210)
-            cy.get('[data-cy="profile-popup"]').should('not.exist')
-            cy.get('[data-cy="profile-popup-content"]').should('not.exist')
-            cy.get(olSelector).click(120, 240)
-            cy.get('[data-cy="profile-popup"]').should('be.visible')
+            cy.get('[data-cy="infobox"]').should('not.be.visible')
         })
         it('opens up the popup again if the user clicks on the feature', () => {
-            cy.get(olSelector).click(360, 210)
-            cy.get(olSelector).click(120, 240)
-            cy.get('[data-cy="profile-popup"]').should('be.visible')
-            cy.get('[data-cy="profile-popup-content"]').should('be.visible')
+            cy.get(olSelector).then(($map) => {
+                let center = $map.outerWidth() / 2
+                cy.get(olSelector).click(360, 210)
+                cy.get(olSelector).click(center, 220)
+                cy.get('[data-cy="infobox"]').should('be.visible')
+            })
         })
         it('closes the profile popup when click on the X button', () => {
-            cy.get('[data-cy="profile-popup-close-button"]').click()
-            cy.get('[data-cy="profile-popup"]').should('not.exist')
-            cy.get('[data-cy="drawing-style-popup"]').should('not.exist')
+            cy.get('[data-cy="infobox-close"]').click()
+            cy.get('[data-cy="infobox"]').should('not.be.visible')
         })
         it('closes the profile popup, while minimized, when the user clicks on the X button', () => {
-            cy.get('[data-cy="profile-popup-minimize-button"]').click()
-            // TODO: find a way to place the profile popup's close button out of the layers copyrights (and then remove the { force: true })
-            // see : https://jira.swisstopo.ch/browse/BGDIINF_SB-2170
```

Comment:
Is https://jira.swisstopo.ch/browse/BGDIINF_SB-2170 fixed by your changes?

Reply:
This was fixed long ago when we moved the profile to `#ap-footer-middle`. But it seems I forgot to update the test back then.